contents/docs/experiments/manual.mdx

```diff
@@ -8,136 +8,314 @@ availability:
+- The impact of offering a free trial period versus a money-back guarantee to determine which approach results in more paying customers.
 
-## Creating an experiment
+A/B tests are also sometimes referred to as "experiments", and this is how we refer to them in the PostHog app.
 
-Creating an experiment starts with a hypothesis you want to test. Let's use an example of whether a visual update to the sign up button will increase clicks for that button.
+## How to create an experiment in PostHog
 
-After clicking on "new experiment" in the Experiments page, we should see a form like this:
+To create a new experiment, go to the [experiments tab](https://app.posthog.com/experiments) in the PostHog app, and click on the "New experiment" button in the top right.
 
-![Create experiment](../../images/features/experiments/create-experiment.png)
+This presents you with a form where you can complete the details of your new experiment:
 
-#### 1. Experiment name, feature flag, and testing variants
+![How to create an experiment in PostHog](../../images/features/experiments/create-experiment.png)
 
-Let's start in the upper left corner by filling out a name, description, and feature flag for our experiment. Fill in a name for your feature flag - it will be created here by default, linked to the experiment rather than existing as its own entity.
+Here's a breakdown of each field in the form:
 
-![Name, feature flag, and variants](../../images/features/experiments/create-experiment-name-variants.png)
+### Feature flag key
 
-We can also include multivariants for our testing changes. For example, instead of running our experiment with just "control" and "test", we can test for the green visual button change or the orange one or our original control.
+Each experiment is backed by a [feature flag](/docs/feature-flags/manual). In your code, you use this feature flag key to check which experiment variant the user has been assigned to. 
 
-#### 2. Participant selection and experiment goal/metric
+> **Advanced:** It's possible to create experiments without using PostHog's feature flags (for example, if you you're a different library for feature flags). For more info, read our docs on [implementing experiments without feature flags](// TODO add link here) 
 
-Next, we want to select our experiment's universe. Here, we've decided that for our sign up button experiment we want to test our hypothesis on users from Asia.
+### Experiment variants
 
-We also want to set our experiment's "goal" now for which metric we want to track. Since we're testing whether the number of sign up button clicks increase or not, we'll use the "Trend" goal to measure aggregate values
+By default, all experiments have a 'control' and 'test' variant. If you have more variants that you'd like to test, you can use this field to add more variants and run multivariant tests. Participants are automatically split equally between variants.
 
```

Comment:
Do you want to add a note on overrides here? "Participants are automatically split equally between variants. If you want specific users in a specific variant, you can set up manual overrides" (and add a link?)

Reply:
Sounds good. I'll add this in the part 3 PR, since I added that question to the FAQ there, so I can link to it there https://github.com/PostHog/posthog.com/pull/6189/files